backend/tests/users/test_tokens.py

```diff
@@ -47,7 +47,8 @@ async def basic_passthrough(self, request: fastapi.Request):
     monkeypatch.setattr(HTTPBasicAuth, "__call__", basic_passthrough)
     token_string = f"{unauthenticated_user.name}:myTestPassword"
     token = base64.b64encode(token_string.encode("ascii"))
-    basic_response = client.post(
+    basic_client = testclient.TestClient(app)
```

Comment:
client is unused as parameter in the function signature.

Reply:
Good catch!